.github/workflows/draft-release.yml

```diff
@@ -61,45 +72,61 @@ jobs:
       - name: Store release tarball as artifact
         uses: actions/upload-artifact@v2
         with:
-          name: tarball
+          name: release-tarball
           path: ${{ env.TARBALL_PATH }}
           if-no-files-found: error
 
+  create-packages:
+    needs: create-release-tarball
+    uses: ./.github/workflows/create-packages.yml
+    with:
+      source-tarball-artifact-name: release-tarball
+      dbld-image-mode: build
+
+  upload-packages:
+    needs: create-packages
+    uses: ./.github/workflows/upload-packages.yml
+    with:
+      pkg-type: stable
+    secrets:
+      azure-sas-token: ${{ secrets.AZURE_SAS_TOKEN }}
+
+  create-draft-release:
+    runs-on: ubuntu-latest
+    needs: [create-release-tarball, upload-packages]
```

Comment:
Yes, I noticed the hover thing, too.

`create-release-tarball` is needed, because we need `outputs` from there, `upload-packages` is needed, so `create-draft-release` does not start until it is finished.

I can add every other step, too, so the hovering is better, but I would prefer the simplicity of the workflow file over the mouse hovering "fix".

Reply:
Maybe I was unclear, sorry. Let me try again.
(of course this is minor thing, but I think it worth it as I got confused about the graph of the jobs)

Jobs in a workflow run parallel by default, you can make them run in a sequence with the `needs` keyword.
Draft-release is now made out of multiple jobs, and they are ordered in a strict sequence.

What I meant earlier is that it should be enough that 1 job only depends on the preceding job.
It's not needed to mention earlier jobs to get their artifacts, AFAIK artifacts are shared in a workflow run (and this is different to GitLab CI I think).
It's not "just a UI thing", but it can be confusing to understand the order in which the jobs run: `needs` show what follows what.
Seeing multiple jobs in a job's `needs` keyword caught my eye as I suddenly got confused about what's running concurrently.

Reply:
`create-release-tarball` is not needed because of its artifact, but because of its `outputs`, see `create-draft-release` -> `Create draft release`, e.g.: `${{ needs.create-release-tarball.outputs.RELEASE_TAG }}`.

I would also be happy, if I could only reference the previous job, but unfortunately it is not possible in this case.

Reply:
Ohhh, my bad. You even emphasized the reason is `outputs`.... Sorry I just didn't parse your whole comment at first.
Thanks for clarifying.